process-document/src/main/kotlin/com/ritense/processdocument/service/CorrelationService.kt

```diff
@@ -0,0 +1,15 @@
+package com.ritense.processdocument.service
+
+import org.camunda.bpm.engine.runtime.MessageCorrelationResult
+
+interface CorrelationService {
```

Comment:
regarding the MessageCorrelationResult i  added it to the sendStartMessage.
regarding the third option you mention do you mean correlating an intermediate catch event? i guess the sendMessageToAll covers it because one or multiple is the same thing

Reply:
No it's not. Correlating one means only correlating one. Correlating all means correlating all. The 4th option describes a correlateAll method, and the third option describes a correlate method. They behave differently.